torch/jit/__init__.py

```diff
@@ -18,6 +18,9 @@
 import numbers
 import collections
 import re
+# sys.path.append('/Users/chunlifu/pytorch/torch/csrc/jit/script')
+# from batch_operators import *
+# import batch_operators
```

Comment:
Should probably remove these before checking in. cc @zdevito  we need to figure out how to fix circular dependencies